### PR TITLE
Change to set custom define ghq command

### PR DESCRIPTION
**Pull request by w1mvy**
Hi! this plugin is nice and helpful for me!

I changed to ghq command can be defined by the user.

## Motivation

ghq v0.13.1 optimized `--vcs` option of `ghq list`.
I only use git, so I want to use this option.

### CODE REVIEW

File: readme.mkd [hunk 1]
```diff
@@ -15,3 +15,11 @@ Latest: https://github.com/Jagua/vim-denite-ghq
...
 
 ```
+
+You can change ghq option as follows example.
+
+```
+# Use vcs option
+call denite#custom#var('ghq', 'command',
+  \ ['ghq', 'list', '--full-path', '--vcs', 'git'])
+```
```
Comment (Jagua) [inline readme.mkd]: Give language identifier `vim` in fenced code block.
And change to Vim's comment command like the following.

    ```vim
    " Use vcs option

**Review by Jagua — CHANGES_REQUESTED** [review]

File: readme.mkd (hunk 1 above)
Comment (w1mvy) [reply readme.mkd]: @Jagua Thanks for your comment, and I fixed it!
https://github.com/Jagua/vim-denite-ghq/pull/1/commits/2c65d5c1f0d68d3443265fc8544add0c447cbce4

Comment (Jagua) [reply readme.mkd]: Please give language identifier `vim` in fenced code block.

    ```vim

Comment (w1mvy) [reply readme.mkd]: Sorry I overlooked it and changed it. https://github.com/Jagua/vim-denite-ghq/pull/1/commits/1e739cbe1b908b4cff7040ec36e226228eeed698

**Comment by Jagua** [comment]
Merged. Thank you for sending Pull Request!

**Comment by w1mvy** [comment]
Thank you for review & merge!!

**Pull request merged**